### PR TITLE
Set1Task4

### LINKED ISSUE #6: Set1Task4: Perform basic operations on numbers

### PR DESCRIPTION
**Pull request by juturul**
Closes #6

### CODE REVIEW

File: Lavanya.C.Sharp/Set1Task4/Program.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+c = a*b;
+Console.WriteLine($"Product of {a} and {b} is {c}");
+Print(++a, --b);
+
+c = a/b;
+Console.WriteLine($"quotient of {a} and {b} is {c}");
+Print(++a, --b);
+
+c = a%b;
+Console.WriteLine($"Reaminder of {a} and {b} is {c}");
```
Comment (ShriHumrudha) [inline Lavanya.C.Sharp/Set1Task4/Program.cs]: typo

File: Lavanya.C.Sharp/Set1Task4/Program.cs [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+Print(++a, --b);
+
+void Print(int a , int b)
```
Comment (ShriHumrudha) [inline Lavanya.C.Sharp/Set1Task4/Program.cs]: how about leveraging the method for the other console.writeline too?

File: Lavanya.C.Sharp/Set1Task4/Program.cs [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+c = a-b;
+Console.WriteLine($"Difference of {a} and {b} is {c}");
+Print(++a, --b);
+
+c = a*b;
+Console.WriteLine($"Product of {a} and {b} is {c}");
+Print(++a, --b);
+
+c = a/b;
+Console.WriteLine($"quotient of {a} and {b} is {c}");
```
Comment (ShriHumrudha) [inline Lavanya.C.Sharp/Set1Task4/Program.cs]: Consistency in the wording case.

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: Lavanya.C.Sharp/Set1Task4/Program.cs (hunk 1 above)
Comment (juturul) [reply Lavanya.C.Sharp/Set1Task4/Program.cs]: Fixed

File: Lavanya.C.Sharp/Set1Task4/Program.cs (hunk 3 above)
Comment (juturul) [reply Lavanya.C.Sharp/Set1Task4/Program.cs]: Fixed

File: Lavanya.C.Sharp/Set1Task4/Program.cs (hunk 2 above)
Comment (juturul) [reply Lavanya.C.Sharp/Set1Task4/Program.cs]: Moved all console.writeline to a single method but I have to use ref parameters

**Review by ShriHumrudha — APPROVED** [review]

**Pull request merged**